src/main/twirl/com/lightbend/lagom/docs/documentation.scala.html

```diff
@@ -12,16 +12,95 @@
+                    return suggestion._highlightResult.hierarchy['lvl' + level].value ;
+                } else {
+                    return displayValue(suggestion, level - 1) ;
+                }
+            };
+            var displaySnippet = function (suggestion) {
+              if(typeof suggestion._snippetResult !== 'undefined'){
+                return '<div class="algolia-docsearch-suggestion--text">' + suggestion._snippetResult.content.value + '</div>'
+              } else {
+                return ''
+              }
+            }
+
+            var wrap = function(content, wrapper){
+              if(typeof wrapper !== 'undefined'){
+                return '<span class='+wrapper+'>' + content + '</span>';
+              } else {
+                return content;
+              }
+            }
+
+            var buildIndexSettings = function(_indexObj, _hitsPerPage, _faceFilters, title, useFooter, wrapper) {
+              var _header = wrap('<span class="algolia-docsearch-suggestion algolia-docsearch-suggestion__main algolia-docsearch-suggestion__secondary algolia-docsearch-suggestion--category-header ">'+title+'</span>', wrapper);
+              var _templates =  {
+                                    header: _header,
+                                    suggestion: function(suggestion) {
+                                        var suggestionHtml = '<div class="algolia-docsearch-suggestion algolia-docsearch-suggestion__secondary" style="white-space: normal;">' +
+                                         '<div class="algolia-docsearch-suggestion--wrapper">' +
+                                            '<div class="algolia-docsearch-suggestion--subcategory-column ">' +
+                                                '<span class="algolia-docsearch-suggestion--subcategory-column-text">' + displayValue(suggestion, 0) + '</span>' +
+                                            '</div>' +
+                                            '<div class="algolia-docsearch-suggestion--content">' +
+                                                '<div class="algolia-docsearch-suggestion--subcategory-inline ">' + displayValue(suggestion, 1) + '</div>' +
+                                                '<div class="algolia-docsearch-suggestion--title algolia-docsearch-suggestion--duplicate-content">' + displayValue(suggestion, 2)  +'</div>' +
+                                                displaySnippet(suggestion) +
+                                            '</div>' +
+                                        '</div>'+
+                                      '</div>';
+                                      return wrap(suggestionHtml, wrapper)
+                                    }
+                                };
+              if(useFooter){
+                _templates['footer'] = '<div class="algolia-docsearch-footer"><a class="algolia-docsearch-footer--logo" href="https://www.algolia.com/docsearch">Algolia</a></div>';
```

Comment:
If Akka and/or Lagom complete with values, but Play does not, then this will not show. I've tried adding the empty templates and that doesn't work if there are multiple datasets and at least one supplies data...it might need to be an override to their template in some fashion.